pkg/network/usm/usm_http2_monitor_test.go

```diff
@@ -1384,6 +1414,11 @@ func headersWithGivenEndpoint(path string) []hpack.HeaderField {
 // testHeaders returns a set of header fields.
 func testHeaders() []hpack.HeaderField { return generateTestHeaderFields(headersGenerationOptions{}) }
 
+// splitTestHeaders returns a set of header fields spited by the given index.
+func splitTestHeaders(index int) ([]hpack.HeaderField, []hpack.HeaderField) {
+	return generateTestHeaderFields(headersGenerationOptions{})[:index], generateTestHeaderFields(headersGenerationOptions{})[index:]
+}
```

Comment:
I don't see a value for this function, just put it inline

Reply:
removed